File: 사칙/src/num2439.java
import java.util.Scanner;
public class num2439 {
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();
        for(int i = 0;i<num;i++){
            for(int k = num;k>i+1;k--){

                System.out.print(" ");
            }
            for(int j = 0 ;j<=i;j++){
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: 사칙/bin/num2439.py
num = int(input())

for i in range(num):
    for k in range(num-1,i,-1):
        print(" ",end="")
    for j in range(0,i+1,1):
        print("*",end="")
    print()

n=int(input())
for i in range(1,n+1):
    print(' '*(n-i)+'*'*i)